src/backend/src/yaksuri_progress.c

```diff
@@ -2963,6 +2963,7 @@ int yaksuri_progress_enqueue(const void *inbuf, void *outbuf, uintptr_t count, y
     if (!subreq) {
         goto fn_exit;
     }
+    subreq->gpudriver_id = id;
```

Comment:
Could you add a comment that "reqpriv->gpudriver_id" may be different between multiple `yaksuri_progress_enqueue` ?

Reply:
I added an according comment.